debug/src/debug.js

```diff
@@ -105,7 +105,7 @@ export function initDebug() {
 					`\n\n${getOwnerStack(vnode)}`
 			);
 		} else if (type != null && typeof type === 'object') {
-			if (type._lastDomChild !== undefined && type._dom !== undefined) {
+			if (type._lastDomChildSibling !== undefined && type._dom !== undefined) {
```

Comment:
Hmm I think I might've messed up this condition here... Need to look more at it

Reply:
Okay, I think the latest change fixes this by using `_children` as the sentinel property.